tf/do/main.tf

```diff
@@ -0,0 +1,220 @@
+provider "digitalocean" {
+    token = var.do_token
+}
+
+resource "random_id" "droplet_id" {
+    byte_length = 8
+}
+
+resource "digitalocean_droplet" "harmony_node" {
+    name        = "node-s${var.shard}-${var.blskey_index}-${random_id.droplet_id.hex}"
+    image       = var.droplet_system_image
+    region      = var.droplet_region
+    size        = var.droplet_size
+    tags        = ["harmony"]
+    # ssh_keys    = [digitalocean_ssh_key.harmony_ssh_key.fingerprint]
+    ssh_keys = [25624276]
```

Comment:
what's this?

Reply:
This has been updated to use a regular ssh key pair.